controller/controller.go

```diff
@@ -240,8 +240,22 @@ func Install(rootDir string, model *model.SystemInstall, options args.Args) erro
 		if val, ok := model.InstallSelected[curr.Name]; ok {
 			wholeDisk = val.WholeDisk
 		}
+
+		dryRun := &storage.DryRunType{CurrentTargetResults: &[]string{}, OtherMediaResults: &[]string{}}
```

Comment:
Let's talk offline about this new need for two sets of results.

Reply:
We decided to have this structure with different field names. Marking as resolved